plugin_interpreter/src/controller_plugin.py

```diff
@@ -252,13 +259,13 @@ def _request_job(self):
             self._update_job(job["id"])
         return job
 
-    def _respond_output(self, job, output):
```

Comment:
API changes usually deprecate over at least 1 sprint.

Suggest leaving _respond_output with a stderr.write that it's now deprecated

Add a task to remove in a future sprint.